polkadot/runtime/test-runtime/src/lib.rs

```diff
@@ -340,6 +340,7 @@ parameter_types! {
 	pub storage SessionsPerEra: SessionIndex = 6;
 	// 28 eras for unbonding (7 days).
 	pub storage BondingDuration: sp_staking::EraIndex = 28;
+	pub storage MaxBondedEras: u32 = (BondingDuration::get() as u32) + 1;
```

Comment:
```suggestion
	pub const MaxBondedEras: u32 = (BondingDuration::get() as u32) + 1;
```

I think all of this are defined as `pub storage` for no reason? not a big deal, but it seems unnecessary. 

`pub storage` crates an onchain storage item for them. `pub const` is a embeded constant.

Reply:
Are you referring to all the parameters here or just to `MaxBondedEras`?